tests/conftest.py

```diff
@@ -3620,7 +3620,7 @@ def measurement_dir(tmp_path):
 
 
 @pytest.fixture()
-def multi_dc_pod(multi_pvc_factory, dc_pod_factory, service_account_factory):
+def multi_dc_pod(multi_pvc_factory, deploy_pod_factory, service_account_factory):
```

Comment:
rename `multi_dc_pod` to `multi_deployment_pods`

Reply:
Resolved with latest commit